CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.16)
 project(ArborX CXX)
 
-find_package(Kokkos 3.1 REQUIRED)
+find_package(Kokkos 3.1 REQUIRED CONFIG)
```

Comment:
Is it required to be a part of the PR? And would you want to restrict `find_package` to only `CONFIG` mode?

Reply:
Required to be part of the PR no.  Yes don't want to look for a find module